### PR TITLE
Fix: M2-630 - Builder - Applet About Page: Images requirements text issues

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/modules/Builder/features/AboutApplet/AboutApplet.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { colorThemeOptions } from './AboutApplet.const';
 const commonUploaderProps = {
   width: 20,
   height: 20,
-  maxFileSize: MAX_FILE_SIZE_1GB,
+  maxFileSize: MAX_FILE_SIZE_5MB,
```
Comment (alex-i-e) [inline src/modules/Builder/features/AboutApplet/AboutApplet.tsx:24]: Can you check if there are  the same restrictions for images in Activity About Page?

File: src/modules/Builder/features/AboutApplet/AboutApplet.tsx
Comment (IhorNesterenko99) [inline src/modules/Builder/features/AboutApplet/AboutApplet.tsx:1]: Add also to Activity About and remove MAX_FILE_SIZE_1GB if no needed

File: src/modules/Builder/features/ActivityAbout/ActivityAbout.tsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ export const ActivityAbout = () => {
           {...commonUploaderProps}
           setValue={(val: string) => setValue(`${fieldName}.splashScreen`, val)}
           getValue={() => watch(`${fieldName}.splashScreen`)}
-          description={t('uploadTransfluent', { size: byteFormatter(MAX_FILE_SIZE_1GB) })}
+          description={t('uploadImg', { size: byteFormatter(MAX_FILE_SIZE_5MB) })}
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/ActivityAbout/ActivityAbout.tsx:95]: why did you change description?

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by alex-i-e — APPROVED** [review]
